src/Wallpaperize.vala

```diff
@@ -23,6 +23,17 @@ public class Wallpaperize.Wallpaperiser : Object {
       }
     }
 
+    public static void get_monitor_geometry() {
+        Gdk.Screen screen = Gdk.Screen.get_default();
```

Comment:
make use of `var` wherever you can :) 

`var screen = Gdk.Screen.get_default ();`